util/caller/resolver.go

```diff
@@ -55,57 +69,39 @@ func NewCallResolver(offset int, numStrip int) *CallResolver {
 		panic("could not look up callsite of NewCallResolver")
 	}
 	fileParts := strings.Split(file, "/")
-	cr.stripPrefix = strings.Join(fileParts[:len(fileParts)-numStrip-1], "/") + "/"
+	if len(fileParts) > numStrip {
+		cr.stripPrefix = strings.Join(fileParts[:len(fileParts)-numStrip-1], "/") + "/"
```

Comment:
this looks too fancy. i think we're always going to to strip everything up to and including the `"cockroach"+string(os.PathSeparator)`